contracts/InterchainProposalExecutor.sol

```diff
@@ -115,6 +115,32 @@ contract InterchainProposalExecutor is IInterchainProposalExecutor, AxelarExecut
         emit WhitelistedProposalSenderSet(sourceChain, sourceSender, whitelisted);
     }
 
+    /**
+     * @dev Withdraw given balance to the recipient.
+     * This function is only callable by the owner.
+     */
+    function withdraw(address recipient, uint256 balance) public onlyOwner {
```

Comment:
not needed since it can be simulated via arbitrary execution. we can add an example/unit test to showcase it instead